### PR TITLE
added V8LazyParseWebpackPlugin, UglifyJsPlugin , async attr , preload bundles

### PR DESCRIPTION
**Pull request by kuldeepkeshwar**
added few things for perf

- async attr
- UglifyJsPlugin
- V8LazyParseWebpackPlugin
- preload bundles

### CODE REVIEW

File: webpack.config.babel.js [hunk 1]
```diff
@@ -89,6 +90,24 @@ module.exports = {
 	],
 
 	plugins: ([
+		new V8LazyParseWebpackPlugin(),
+		new webpack.optimize.UglifyJsPlugin({
```
Comment (developit) [inline webpack.config.babel.js]: This is actually going to run Uglify in development mode, and twice in production mode.
We run `webpack -p` for production, which include UglifyJSPlugin, DedupePlugin by default.

**Comment by kuldeepkeshwar** [comment]
I think, we need to update node/npm version for travis.
@developit rigth ?

**Comment by developit** [comment]
Other than the uglify thing I think this is looking good! For uglify, either we'll need to move it into the NODE_ENV==='production' section, add the two missing optimization.* plugins and remove `-p` from the npm script, or strip it out for now. Would be curious to see the effect on output size over the uglify defaults already applied by `webpack -p`

File: webpack.config.babel.js (hunk 1 above)
Comment (kuldeepkeshwar) [inline webpack.config.babel.js]: sure, will move it to the conditional part

**Comment by developit** [comment]
@kuldeepkeshwar RE node version - it'd be nice to keep node 4 / npm 2 compatibility. It looks like theres a bug in LazyParseWebpackPlugin that is causing the error you're seeing: TheLarkInn/LazyParseWebpackPlugin#6.

**Comment by kuldeepkeshwar** [comment]
@developit somehow it worked on my local with `V8LazyParseWebpackPlugin` 
```
KuldeepMac:preact-boilerplate kuldeepkeshwar$ node -v
v6.9.1
KuldeepMac:preact-boilerplate kuldeepkeshwar$ npm -v
3.10.8
```

**Comment by developit** [comment]
Yup, I think it just doesn't support Node 4 and prior.

**Comment by kuldeepkeshwar** [comment]
commented V8LazyParseWebpackPlugin

**Comment by developit** [comment]
@kuldeepkeshwar nice, that preload stuff is good.

**Comment by developit** [comment]
Think we're good to merge?

**Comment by kuldeepkeshwar** [comment]
Yes :-)

**Comment by kuldeepkeshwar** [comment]
I will un-comment V8LazyParseWebpackPlugin once the issue with node v4 get resolved

**Comment by developit** [comment]
I've submitted a PR that should hopefully fix the V8LazyParseWebpackPlugin issue:
TheLarkInn/LazyParseWebpackPlugin#8

**Comment by kuldeepkeshwar** [comment]
@developit
as TheLarkInn/LazyParseWebpackPlugin#8 is merged now, we can use V8LazyParseWebpackPlugin, I'm updating the PR

**Comment by kuldeepkeshwar** [comment]
updated PR :-)

**Comment by developit** [comment]
Awesome, thanks!

**Pull request merged**